qchem/pennylane_qchem/qchem.py

```diff
@@ -639,7 +639,51 @@ def sd_excitations(n_electrons, n_orbitals, delta_sz=0):
              if (sz[p] + sz[q] - sz[r] - sz[s]) == delta_sz
     ]
 
-    return ph, pphh    
+    return ph, pphh
+
+
+def hf_state(n_electrons, m_spin_orbitals):
+    r"""Generates the occupation-number vector representing the Hartree-Fock (HF)
+    state of :math:`N` electrons in a basis of :math:`M` spin orbitals.
+
+    The many-particle wave function in the HF approximation is a `Slater determinant
+    <https://en.wikipedia.org/wiki/Slater_determinant>`_. In Fock space, a Slater determinant
+    is represented by the occupation-number vector:
+
+    .. math:
+        \vert {\bf n} \rangle = \vert n_1, n_2, \dots, n_M \rangle,
+
+        n_i = \left\lbrace \begin{array}{ll} 1 & i \leq N \\ 0 & i > N \end{array} \right..
+
+    **Example**
+
+    >>> init_state = hf_state(2, 6)
+    >>> print(init_state)
+    [1 1 0 0 0 0]
+
+    Args:
+        n_electrons (int): number of active electrons
+        m_spin_orbitals (int): number of active **spin-orbitals**
+
+    Returns:
+        array: NumPy array containing the vector :math:`\vert {\bf n} \rangle`
+    """
+
+    if not n_electrons > 0:
+        raise ValueError(
+            "The number of active electrons has to be > 0; got 'n_electrons' = {}"
```

Comment:
A similar error messages might be added for cases with m_spin_orbitals <= 0 (or maybe 1).

Reply:
@soranjh, just realized that this case is already covered by he previous checks. 
For example: 
- if ``n_electrons > 0`` and ``m_spin_orbitals <= 0`` the function raises an error as `` n_electrons > m_spin_orbitals``.

- if ``n_electrons < 0`` and ``m_spin_orbitals <= 0`` the function raises an error as `` n_electrons >=0 ``.